src/services/quandl-service.js

```diff
@@ -45,6 +70,16 @@
     }
 
     function processResponse(json) {
+        if (!json || json.quandl_error) {
+            return {
+                success: false,
+                error: {
+                    code: json.quandl_error.code,
+                    message: json.quandl_error.message
+                }
+            };
+        }
```

Comment:
I'm not sure what this is doing -- `processResponse` previously was not returning anything, but was appending a processed response to the input JSON object. Where this method is called no variable is assigned to the return value, either.